### PR TITLE
Some cleanup to the manage_competition_spec.

### PR DESCRIPTION
**Pull request by jfly**
I didn't leave #954 for comments, because I wanted to fix the bug in production immediately. I'll leave this PR open to gather comments on how to have better written the manage_competition_spec.

### CODE REVIEW

File: WcaOnRails/spec/features/manage_competition_spec.rb [hunk 1]
```diff
@@ -17,6 +20,7 @@
       fill_in "Name", with: "New Comp 2015"
 
       click_button "Create Competition"
+      expect(page).to have_content "Successfully created new competition!"
 
       expect(Competition.all.length).to eq 1
```
Comment (jonatanklosko) [inline WcaOnRails/spec/features/manage_competition_spec.rb]: According to Tim's thoughts this line should be removed, right?

File: WcaOnRails/spec/features/manage_competition_spec.rb [hunk 2]
```diff
@@ -31,6 +35,7 @@
 
       expect(page).to have_button('Create Competition')
       click_button "Create Competition"
+      expect(page).to have_content "Successfully created new competition!"
 
       expect(Competition.all.length).to eq 2
```
Comment (jonatanklosko) [inline WcaOnRails/spec/features/manage_competition_spec.rb]: Same as above.

File: WcaOnRails/spec/features/manage_competition_spec.rb (hunk 1 above)
Comment (jfly) [inline WcaOnRails/spec/features/manage_competition_spec.rb]: That was not my understanding of @timhabermaas's comment [here](https://github.com/thewca/worldcubeassociation.org/pull/954#discussion_r83903305). The race was that we'd click the button and then immediately check the database... by waiting for the resulting page to load first, we know that the server has already written to the database before we go and look at the database.

**Comment by larspetrus** [comment]
LGTM

**Pull request merged**